README.md

```diff
@@ -17,216 +48,361 @@ See also:
-You should get a result similar to this example:
 
-   ```json
-   {"repositories":["keda-manager-dev-local","unsigned/component-descriptors/kyma-project.io/module/keda"]}
-   ```
-6. Inspect the generated module template.
+6. Deploy.
 
-> **NOTE:** The following sub-steps are temporary workarounds.
-
-Edit `template.yaml` and:
-- change `target` to `control-plane`
-
-   ```yaml
-   spec:
-    target: control-plane
-    ```
-> **NOTE:** This is required in the single-cluster mode only.
+   ```bash
+   make deploy
+   ```
 
-- change the existing repository context in `spec.descriptor.component`:
+7. Verify if keda manager is deployed
 
-   ```yaml
-   repositoryContexts:      
-     - baseUrl: k3d-kyma-registry.localhost:5000/unsigned
-       componentNameMapping: urlPath
-       type: ociRegistry
+   ```bash
+   kubectl get deployments -n kyma-system       
+   NAME                             READY   UP-TO-DATE   AVAILABLE   AGE
+   operator-keda-manager            1/1     1            1           1m
    ```
 
-> **NOTE:** Because Pods inside the k3d cluster use the docker-internal port of the registry, it tries to resolve the registry against port 5000 instead of 5001. K3d has registry aliases, but `module-manager` is not part of k3d and does not know how to properly alias k3d-kyma-registry.localhost:5001.
 
-7. Install the modular Kyma on the k3d cluster.
+### How to use Keda Manager to instal Keda
 
-> **NOTE** This installs the latest versions of `module-manager` and `lifecycle-manager`.
+Keda manager installs KEDA based on watched Keda Custom Resources (CR):
 
-Use the `--template` flag to deploy the Keda module manifest from the beggining, or apply it using kubectl later.
+- Apply Keda CR (sample) to have Keda installed.
 
    ```bash
-   kyma alpha deploy --template=./template.yaml
+   kubectl apply -f config/samples/operator_v1alpha1_keda_k3d.yaml
    ```
-Kyma installation is ready, but the module is not yet activated.
 
-   ```bash
-   kubectl get kymas.operator.kyma-project.io -A
-   ```
-You should get a result similar to the following example:
+   This will make keda manager busy and after a while you will have Keda installed and you shall see it's workloads:
 
    ```bash
-   NAMESPACE    NAME           STATE   AGE
-   kcp-system   default-kyma   Ready   71s
+   NAME                             READY   UP-TO-DATE   AVAILABLE   AGE
+   operator-keda-manager            1/1     1            1           3m
+   keda-manager                     1/1     1            1           3m
+   keda-manager-metrics-apiserver   1/1     1            1           3m
    ```
 
-Keda Module is a known module, but not yet activated.
+   Now you can use KEDA to scale workloads on the k8s cluster. Please check the [demo application](docs/keda-demo-application.md).
 
-   ```bash
-   kubectl get moduletemplates.operator.kyma-project.io -A 
-   ```
-
-You should get a result similar to the following example:
+- Remove Keda CR to have Keda uninstalled.
```

Comment:
```suggestion
- Remove Keda CR to uninstall Keda.
```

Reply:
I actually think the passive voice is better here as its not the user uninstalling it, but manager reacting to CR removal